homeassistant/components/zha/__init__.py

```diff
@@ -90,6 +90,17 @@
     """
 
     hass.data[DATA_ZHA] = hass.data.get(DATA_ZHA, {})
```

Comment:
```py
zha_data = hass.data.setdefault(DATA_ZHA, {})
config = zha_data.get(DATA_ZHA_CONFIG, {})
```

Reply:
Fixed in https://github.com/home-assistant/core/pull/32959/commits/4b0bc837e0a44f80fe0fba0c042ca129e6083c54